app/code/core/Mage/Adminhtml/Model/Config/Data.php

```diff
@@ -143,12 +143,16 @@ public function save()
                     }
                 }
 
-                /**
-                 * Get field backend model
-                 */
-                $backendClass = (isset($fieldConfig->backend_model)) ? $fieldConfig->backend_model : false;
-                if (!$backendClass) {
-                    $backendClass = 'core/config_data';
+                // Get field backend model
+                $backendClass = 'core/config_data';
+                if (isset($fieldConfig->backend_model)) {
+                    $backendClass = (string)$fieldConfig->backend_model;
+                } elseif (isset($fieldConfig->frontend_type)) {
+                    switch ((string)$fieldConfig->frontend_type) {
+                        case 'color':
+                            $backendClass = 'adminhtml/system_config_backend_color';
+                            break;
+                    }
```

Comment:
Careful that this seems to change behavior. What if `<backend_model>` exists, but is an empty string? Before, we would have `core/config_data` but now an exception

Reply:
Actually, I'm wrong on it changing behavior, because `$fieldConfig->backend_model` will be a `SimpleXMLElement`, which is never falsy.

Reply:
ye.. I thought about that but said "who the heck has a `<backend_model></backend_model>` tag... and I liked the way this code looks more... should I revert this few lines?

Reply:
> should I revert this few lines?

@fballiano wait a few minutes ... PR/Suggestion ready in a few minutes

Reply:
@fballiano Did you see my previous comment about the other place we may need this logic at? https://github.com/justinbeaty/magento-lts/commit/684debc1649ae1a2fa4999d4c5382c85fc81b6d3

Reply:
@justinbeaty unfortunately i saw it too late. Your're right.